src/frontend/package.json

```diff
@@ -3,17 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@types/jest": "24.0.11",
-    "@types/lodash": "4.14.123",
+    "@types/jest": "24.0.12",
+    "@types/lodash": "4.14.126",
     "@types/node": "11.12.2",
-    "@types/react": "16.8.10",
-    "@types/react-dom": "16.8.3",
+    "@types/react": "16.8.17",
+    "@types/react-dom": "16.8.4",
```

Comment:
It's really not obvious how to pick the right version. You're supposed to match the major.minor part libraries' versions, but that generally doesn't work.